src/pages/Withdrawals/index.tsx

```diff
@@ -0,0 +1,753 @@
+                    during the next sweep."
+                  />
+                </Text>
+              </section>
+            </section>
+          </section>
+
+          <section>
+            <Anchor to="#reward-payments" id="reward-payments">
+              <SubSectionTitle level={3} className="mb10">
+                <FormattedMessage defaultMessage="Reward payments (partial withdrawals)" />
+              </SubSectionTitle>
+            </Anchor>
+            <Text className="mb10">
+              <FormattedMessage
+                defaultMessage="A {rewardPayment} is processed when an active validator has a maxed out
+                effective balance of 32, and has a total balance over 32. A single validator account cannot
+                utilize more than 32 ETH to contribute towards its stake, and thus these accounts will have
+                any extra balance skimmed off in the form of a withdrawal operation."
+                values={{ rewardPayment: <strong>reward payment</strong> }}
+              />
+            </Text>
+            <Text className="mb10">
+              <FormattedMessage defaultMessage="These are also referred to as “partial withdrawals” as the remaining 32 ETH stays locked and staked." />
+            </Text>
+            <Alert variant="warning" className="mt30">
+              <span role="img" aria-label="note">
+                📝
+              </span>{' '}
+              <FormattedMessage defaultMessage="It is not possible to manually request specific amounts of ETH to be withdrawn" />
+            </Alert>
+          </section>
+
+          <section>
+            <Anchor to="#how-to-exit" id="how-to-exit">
+              <SubSectionTitle level={3} className="mb10">
+                <FormattedMessage defaultMessage="How to exit from staking (full withdrawals)" />
+              </SubSectionTitle>
+            </Anchor>
+            <Text className="mb10">
+              <FormattedMessage
+                defaultMessage="A {fullWithdrawal} is processed for any inactivated accounts that are no longer considered to be staking, that has fully exited from its validator responsibilities. For an account to be processed as a full withdrawal, it must first complete the process of exiting from its validator responsibilities. "
+                values={{ fullWithdrawal: <strong>full withdrawal</strong> }}
+              />
+            </Text>
+
+            <section className="actionable">
+              <Heading level={4} className="mb10">
+                <FormattedMessage defaultMessage="To exit staking and fully withdrawal your entire remaining balance, you must do two things:" />
+              </Heading>
+              <ol>
+                <li>
+                  <FormattedMessage
+                    defaultMessage="Make sure you’ve updated your withdrawal credentials and have a withdrawal
+                    address registered to this account"
```

Comment:
Just an attempt to add clarity that "updating your withdrawal credentials" means that you've provided a withdrawal address... I can see how it may look like a second task though.
What about:
"Make sure you’ve updated your withdrawal credentials with an execution withdrawal address"

```suggestion
```